python/paddle/fluid/layers/control_flow.py

```diff
@@ -1018,8 +1018,17 @@ def body(i, ten):
         return loop_vars
 
     while_loop_block = While(pre_cond, is_test, name)
+    with_mutable_vars = assert_with_mutable_vars(loop_vars)
     with while_loop_block.block():
-        output_vars = body(*loop_vars)
+        # If a variable with mutable type is included in vars, like `dict/list`,
+        # modifying them in the body function will cause origin variable be modified
```

Comment:
Grammatical error: "cause origin variable be modified" to "cause origin variable to be modified"

Reply:
Thanks, fixed.